### PR TITLE
Ben 9000

### PR DESCRIPTION
**Pull request by benoah**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (mannuelf) [inline .vscode/settings.json:2]: please delete this file and push your changes.

File: 1_space_odyssey/index.html [hunk 2]
```diff
@@ -9,8 +9,8 @@
     <title>2020: A Space Odyssey</title>
 </head>
 <body class="fade-in">
-    
-    <script src="./js/mannuel-9000/app.js"></script>
+<script src="js/mannuel-9000/ben-9000/app.js"></script>    
+  
     <!-- include your javascript file here -->
```
Comment (mannuelf) [inline 1_space_odyssey/index.html]: please put my JS include back and add yours after mine ;-)

File: 1_space_odyssey/js/mannuel-9000/ben-9000/app.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = true;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
```
Comment (mannuelf) [inline 1_space_odyssey/js/mannuel-9000/ben-9000/app.js]: please do exercise 2. as well.

its this one: https://github.com/mannuelf/feu1_programming_foundations/blob/master/1_space_odyssey/js/mannuel-9000/app.js#L111

File: 1_space_odyssey/js/mannuel-9000/ben-9000/app.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+var isAlive = true;
```
Comment (mannuelf) [inline 1_space_odyssey/js/mannuel-9000/ben-9000/app.js]: please put your named folder into the root of the JS folder, outside out my name folder.
so the path will look like this:
`js/ben-9000/app.js`

File: 1_space_odyssey/index.html (hunk 2 above)
Comment (benoah) [reply 1_space_odyssey/index.html]: Did just fix it, help from you:)

File: 1_space_odyssey/js/mannuel-9000/ben-9000/app.js (hunk 4 above)
Comment (benoah) [reply 1_space_odyssey/js/mannuel-9000/ben-9000/app.js]: we did that to:)

**Comment by benoah** [comment]
I think it is correct.

**Review by mannuelf — APPROVED** [review]

**Pull request merged**